src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -379,6 +380,19 @@ services:
       - name: tactician.handler
         command: PrestaShop\PrestaShop\Core\Domain\Product\Command\RemoveAllRelatedProductsCommand
 
+  prestashop.adapter.product.query_handler.get_packed_products_handler:
```

Comment:
This one is a duplicate The handler is already defined in `_pack.yml`

Reply:
I have to move it in pack.yml